include/llvm/IR/DiagnosticPrinter.h

```diff
@@ -55,6 +55,7 @@ class DiagnosticPrinter {
 
   // Other types.
   virtual DiagnosticPrinter &operator<<(const SMDiagnostic &Diag) = 0;
+  virtual DiagnosticPrinter &operator<<(std::ios_base &(*iomanip)(std::ios_base&)) = 0;
```

Comment:
The convention is to wrap changes to existing LLVM code base with `// HLSL Change`.

Applies to the following cases.

Reply:
Hmm. There are some clear cases where I missed this, but it might get a bit ridiculous if I go too far with it. Should I put it on every line I changed and around multiline changes including those I just added something to or should I presume that certain corresponding changes can be inferred?

For example, if I comment that the new member variable is an HLSL addition, do I still need to mark every reference of it? Or can it be considered a given?

Happy to go either way. Just don't want to do too much or too little.